server/src/main/java/com/paypal/selion/grid/AppiumSpawner.java

```diff
@@ -41,19 +41,19 @@ public AppiumSpawner(String[] args, ProcessLauncherOptions options) {
         setType(InstanceType.APPIUM);
     }
 
-    public static final void main(String[] args) {
+    public static void main(String[] args) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2103039166/issues/source?bid=2928425&fileBranchId=3110989#l44)